docs/connectors/kafka-connect-connector.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "Kafka Connect Connectors with Conduit"
+sidebar_position: 8
+---
+
+# Using Kafka Connect Connectors with Conduit
+
+The [Conduit Kafka Connect Wrapper connector](https://github.com/ConduitIO/conduit-kafka-connect-wrapper) is a special
+connector that allows you to use [Kafka Connect](https://docs.confluent.io/platform/current/connect/index.html) 
+connectors with Conduit. Conduit doesn't come bundled with Kafka Connect connectors, but you can use it to bring any
+Kafka Connect connector with Conduit.
+
+This connector gives you the ability to:
+
+- Easily migrate from Kafka Connect to Conduit.
+- Remove Kafka as a dependency to move data between data infrastructure.
+- Leverage a datastore if Conduit doesn't have a native connector.
+
+Since the Conduit Kafka Connect Wrapper itself is written in Java, but most of Conduit's connectors are written in Go,
+it also serves as a good example of the flexibility of the Conduit Plugin SDK.
+
+Let's begin.
+
+## How it works
+
+To use the Kafka Connect wrapper connector, you'll need to:
+
+1. Clone the [`conduit-kafka-connect-wrapper`](https://github.com/ConduitIO/conduit-kafka-connect-wrapper) repository.
+1. Build the Connector JAR.
```

Comment:
A release can also be downloaded, so maybe we should mention that as the preferred option.